uicontext.go

```diff
@@ -193,7 +193,7 @@ func (c *uiContext) updateOffscreen() {
 	// scale. This is fine since ebiten.ScreenScale will be deprecated.
 }
 
-func (c *uiContext) setClearingScreenSkipped(skipped bool) {
+func (c *uiContext) setScreenClearedEveryFrame(skipped bool) {
```

Comment:
Rename `skipped`.

Reply:
Not updated yet?

Reply:
I `skipped` this one by accident my bad.